apps/web/src/app/admin-console/organizations/members/services/organization-user/organization-user.service.ts

```diff
@@ -66,4 +65,33 @@ export class OrganizationUserService {
       }),
     );
   }
+
+  bulkConfirmUsers(
+    organization: Organization,
+    userIdsWithKeys: { id: string; key: string }[],
+  ): Observable<ListResponse<OrganizationUserBulkResponse>> {
+    const encryptedCollectionName$ = this.getEncryptedDefaultCollectionName$(organization);
+
+    return encryptedCollectionName$.pipe(
```

Comment:
⛏️  we can inline this variable:
```suggestion
    return this.getEncryptedDefaultCollectionName$(organization).pipe(
```